contrib/platform/test/com/sun/jna/platform/win32/COM/util/ConvertTest.java

```diff
@@ -0,0 +1,274 @@
+package com.sun.jna.platform.win32.COM.util;
+
+import com.sun.jna.Pointer;
+import com.sun.jna.platform.win32.OaIdl.DATE;
+import com.sun.jna.platform.win32.OaIdl.VARIANT_BOOL;
+import com.sun.jna.platform.win32.Ole32;
+import com.sun.jna.platform.win32.Variant;
+import com.sun.jna.platform.win32.Variant.VARIANT;
+import com.sun.jna.platform.win32.WTypes.BSTR;
+import com.sun.jna.platform.win32.WinDef.BOOL;
+import com.sun.jna.platform.win32.WinDef.BYTE;
+import com.sun.jna.platform.win32.WinDef.CHAR;
+import com.sun.jna.platform.win32.WinDef.LONG;
+import com.sun.jna.platform.win32.WinDef.SHORT;
+import java.util.Date;
+import org.junit.AfterClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import org.junit.BeforeClass;
+
+// Untested: IDispatch
+// Untested: Proxy
+public class ConvertTest {
+
+    private static Factory fact;
+
+    @BeforeClass
+    public static void init() {
+        Ole32.INSTANCE.CoInitializeEx(Pointer.NULL, Ole32.COINIT_MULTITHREADED);
+        fact = new Factory();
+    }
+
+    @AfterClass
+    public static void destruct() {
+        fact.disposeAll();
+        fact = null;
+        Ole32.INSTANCE.CoUninitialize();
+    }
+
+    @Test
+    public void convertVariant() {
+        VARIANT testValue = new Variant.VARIANT(42);
+        VARIANT resultVariant = Convert.toVariant(testValue);
+        assertSame(testValue, resultVariant);

```

Comment:
Even though the _@Test_ annotation no longer requires this, it still customary to use the _testXXX_ prefix - e.g., _testConvertVariant_.